### PR TITLE
[Last.fm] Update ruleset

### PR DESCRIPTION
**Pull request by synthtech**

### CODE REVIEW

**Comment by J0WI** [comment]
@synthtech please rebase you PR against current master.

**Comment by synthtech** [comment]
@J0WI rebased

File: src/chrome/content/rules/Last.fm.xml [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
 	CDN buckets:
+		- lastfm-img2.akamaized.net
+
+	Mismatch:
+		- m.cn.last.fm
+		- originals.last.fm
+		- trends.last.fm
+		- cdn.lst.fm
+	
+	Redirect:
```
Comment (J0WI) [inline src/chrome/content/rules/Last.fm.xml]: As long as they don't result in a redirect loop (redirect to HTTP) you should keep them as targets.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**